src/pages/tutor-profile/TutorProfile.jsx

```diff
@@ -41,6 +41,13 @@ const TutorProfile = () => {
     defaultResponse: defaultResponses.array
   })
 
+  useEffect(() => {
+    const tutorVideo = response?.videoLink?.tutor
+    if (!loading && tutorVideo !== null && tutorVideo !== '') {
```

Comment:
Why can't we simplify it?
```suggestion
    if (!loading && tutorVideo) {
```

Reply:
`!loading && tutorVideo` will be false if loading is true, or if `response?.videoLink?.tutor is undefined, null, or an empty string.` This is because JavaScript treats `undefined, null, and "" as "falsy"` values.

Reply:
we need to check for both of these values

Reply:
```
const loading = false;
let tutorVideo;
```
// It just console.log, but if you bring this code to condition it will return true or false
// Case 1: tutorVideo is undefined
```
tutorVideo = undefined;
console.log(!loading && tutorVideo); // Output: false
```

// Case 2: tutorVideo is null
```
tutorVideo = null;
console.log(!loading && tutorVideo); // Output: false
```

// Case 3: tutorVideo is an empty string
```
tutorVideo = '';
console.log(!loading && tutorVideo); // Output: false
```

// Case 4: tutorVideo is a non-empty string
```
tutorVideo = 'some value';
console.log(!loading && tutorVideo); // Output: 'some value'
```

Reply:
![image](https://github.com/ita-social-projects/SpaceToStudy-Client/assets/101183408/710aca8d-1fc8-425d-9335-65c14e1c091d)
fixed

Reply:
good job!